### PR TITLE
[Maps - Tracking] Review request for Common to add version preview/1.0 [Hub Generated] 

### LINKED ISSUE #20297: [Maps - Tracking] API Review
<div data-wrapper="true" style="font-size:9pt;font-family:'Segoe UI','Helvetica Neue',sans-serif;"><div>This meeting was created by Igor Shishkin.</div>
<div>&nbsp;</div>
<div>It will be used to Track the conversation in the informational Session for the Azure Maps Service.</div>
<div>&nbsp;</div>
<div>Detailed meeting information and documents provided can be accessed&nbsp;<a href="https://web.powerapps.com/apps/e41c69de-ef96-44ba-b287-0e012e7c13ae?view=edit&event=c3c76051-1a1f-ed11-b83e-0022480ae914">here</a></div></div>

### PR DESCRIPTION
**Pull request by specialforest**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/specialforest/azure-rest-api-specs/dev-maps-Common-09-01-2022-preview...main/).
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [x] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document: I need these changes as part of https://microsoft.sharepoint.com/:w:/t/Azure-Maps/EQaM-9d6n2FNvxvZpqUJhAgBvANwf5GksWTC5wK6wOhADA?e=tCYbNm
* Previous Open API Doc: NA
* Updated paths: none

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Comment by specialforest** [comment]
> The swagger JSON file is not referenced from the readme file.
readme: [specification/maps/data-plane/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/2b888c4908999f86856446dee444b60fb4677c5a/specification/maps/data-plane/readme.md)
json: [Common/preview/2022-09-01-preview/common.json](https://github.com/Azure/azure-rest-api-specs/blob/2b888c4908999f86856446dee444b60fb4677c5a/specification/maps/data-plane/Common/preview/2022-09-01-preview/common.json)

This is expected because common.json is only referenced from other Swagger specs.

File: specification/maps/data-plane/Common/preview/2022-09-01-preview/common.json [hunk 1]
```diff
@@ -158,9 +158,8 @@
     },
     "BoundingBox": {
       "x-ms-client-name": "boundingBox",
-      "description": "Bounding box. Projection used - EPSG:3857. Format : 'minLon, minLat, maxLon, maxLat'.",
+      "description": "Bounding box. Projection used - EPSG:3857. Please refer to [RFC 7946](https://datatracker.ietf.org/doc/html/rfc7946#section-5) for details.",
       "type": "array",
-      "readOnly": true,
```
Comment (weidongxu-microsoft) [inline specification/maps/data-plane/Common/preview/2022-09-01-preview/common.json]: I see removing of the `readOnly` here. I assume it does not have effect, as `readOnly` should be in property, not in type.

**Review by markweitzel — APPROVED** [review]
Reviewed in the stewardship board via #20297 -- all looks good!

**Review by weidongxu-microsoft — APPROVED** [review]
LGTM.

Only change since last is adding `bbox` in `GeoJsonObject`.

Avocado suppressed in this PR. The missing reference will be solved in https://github.com/Azure/azure-rest-api-specs/pull/20245

**Pull request merged**